### PR TITLE
[openscap] update to 1.3.7

### LINKED ISSUE #27328: [openscap] update to 1.3.6
https://github.com/OpenSCAP/openscap/releases

### PR DESCRIPTION
**Pull request by jiayuehua**
Fixes #27328
coauthored by @JonLiu1993 

- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [ ] ~The "supports" clause reflects platforms that may be fixed by this new version~
- [ ] ~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
- [ ] ~Any patches that are no longer applied are deleted from the port's directory.~
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/openscap/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "openscap",
-  "version": "1.3.1",
-  "port-version": 4,
+  "version": "1.3.6",
```
Comment (Adela0814) [inline ports/openscap/vcpkg.json:3]: The latest version is 1.3.7, please update to [1.3.7](https://github.com/OpenSCAP/openscap/releases/tag/1.3.7). Thanks.

File: ports/openscap/portfile.cmake [hunk 2]
```diff
@@ -1,11 +1,12 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO OpenSCAP/openscap
-    REF 3a4c635691380fa990a226acc8558db35d7ebabc #1.3.1
-    SHA512 12681d43b2ce067c5a5c4eb47e14d91d6e9362b1f98f1d35b05d79ad84c7ee8c29f438eaae8e8465033155e500bb0f936eb79af46fab15e4a07e03c6be8b655d
+    REF 3eccd5c064b9b152fe14a95b3534c60e003be17f #1.3.6
```
Comment (Adela0814) [inline ports/openscap/portfile.cmake:4]:
```suggestion
    REF $VERSION
```

File: ports/openscap/portfile.cmake [hunk 3]
```diff
@@ -45,4 +46,4 @@ vcpkg_copy_pdbs()
 vcpkg_fixup_pkgconfig()
 
 #Handle copyright
-file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
\ No newline at end of file
+file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```
Comment (Adela0814) [inline ports/openscap/portfile.cmake:49]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
```

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

**Comment by jiayuehua** [comment]
@Adela0814 I have summit my change.

**Comment by Adela0814** [comment]
@jiayuehua  I encountered an error while testing the feature.
Repro step:` .\vcpkg.exe install openscap[*]:x64-windows`

Error message:
```
E:\4\vcpkg\buildtrees\openscap\src\1.3.7-278c0d1b45.clean\src\OVAL\probes\crapi\digest.c(30): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory
E:\4\vcpkg\buildtrees\openscap\src\1.3.7-278c0d1b45.clean\src\OVAL\probes\crapi\md5.c(30): fatal error C1083: Cannot open include file: 'sys/mman.h': No such file or directory
```
[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/11733262/install-x64-windows-dbg-out.log)

**Review by Adela0814 — APPROVED** [review]

**Comment by Adela0814** [comment]
Tested all feature on the following triplets:

- x64-windows
- x64-windows-static
- x86-windows

File: ports/openscap/vcpkg.json [hunk 4]
```diff
@@ -19,26 +18,12 @@
...
-    "tests": {
-      "description": "build with tests"
-    },
-    "util": {
-      "description": "build available utils",
-      "dependencies": [
-        {
-          "name": "getopt-win32",
-          "platform": "windows"
-        }
```
Comment (dg0yt) [inline ports/openscap/vcpkg.json:40]: Why did the PR remove the `util` feature? It may break users of the port.

File: ports/openscap/vcpkg.json [hunk 5]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "openscap",
-  "version": "1.3.1",
-  "port-version": 4,
+  "version": "1.3.7",
   "description": "The oscap program is a command line tool that allows users to load, scan, validate, edit, and export SCAP documents.",
```
Comment (dg0yt) [inline ports/openscap/vcpkg.json:4]: And the port is even described as "command line tool".

File: ports/openscap/vcpkg.json (hunk 4 above)
Comment (jiayuehua) [reply ports/openscap/vcpkg.json:40]: #31982

**Pull request merged**